app/models/setting.rb

```diff
@@ -83,4 +88,16 @@ def self.invoice_reminder_in_days
   def self.wishlist_size
     Setting.find_by(code: :wishlist_size).value.to_i
   end
+
+  class << self
```

Comment:
Why is new class needed? I'm pretty sure if you write helper functions under `private` in the same Setting class they work as well